test/jsons/jsonable.json

```diff
@@ -0,0 +1,7 @@
+{
+    "num" : "#num", 
```

Comment:
Dont use json that generates invalid dart. These values contain reserved keywords. Preferably use something like `numValue`